source/includes/code-examples/FindManyAsync.cs

```diff
@@ -0,0 +1,96 @@
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+using MongoDB.Bson.Serialization.Conventions;
+using MongoDB.Driver;
+using MongoDB.Driver.Linq;
+
+namespace CSharpExamples.UsageExamples;
+
+public class FindManyAsync
+{
+    private static IMongoCollection<Restaurant> _restaurantsCollection;
+    private static string _mongoConnectionString = "<Your MongoDB URI>";
```

Comment:
To simplify testing, we can probably create a file like `app.config` (similarly to the `.envrc` used for CSFLE) that each dev can use to set their own MongoDB URI.

Reply:
I think this would be fine for the devs to add to their environments for testing, but my concern with adding it to the "full example" code shown in the repo would be that it would add an extra file that the user would need to add on their own if they are copying these examples. It might make things more complex than necessary if they are just trying to run a quick sample.